src/Mapper/ObjectMapper.php

```diff
@@ -21,7 +21,8 @@ final class ObjectMapper
     public function __construct()
     {
         $this->mappers = [
-            new RequestToRequestMapper(),
+            new PsrRequestToRequestMapper(),
```

Comment:
We might consider doing discovery on mappers in the future. Just a thought

Reply:
A good thought. 👍